### PR TITLE
Rebase and fix-up overall changes

### PR DESCRIPTION
**Pull request by atif-konasl**
**What type of PR is this?**
> Feature

**What does this PR do? Why is it needed?**
This PR contains all the changes in `vanguard` consensus engine. All features which are implemented according to our lukso mainnet architecture are existing in this PR. Main features are remarked below:

1. `pandora` client package for communicating with pandora from `validator` client.
2.  `orchestrator` client package from communicating with orchestrator from `vanguard` client
3. orchestrator consensus verification layer in `vanguard` in block verification phase.
4. introduced several new flags.
5. compatible `pow` service for genesis validators.

**Other notes for review**
This PR candidate branch is currently running in ephemeral l15 testnet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,11 @@
-# Vanguard
```
Comment (mxmar) [inline README.md:1]: I think we should leave this change and add actual # Prysm below :)

**Comment by blazejkrzak** [comment]
This PR is too big for me to fully review it

**Review by blazejkrzak — APPROVED** [review]

**Review by mxmar — APPROVED** [review]
I approve, but before `staging` and `production` we need to consider to write own `README` and `CONTRIBUTION` files for example, right? This PR was already tested against GH actions workflow (k8s/helm) and network is working and progressing from Friday. To many files in one PR to handle by human :)

Before clicking Merge button please solve all conversations :) GJ :+1:

**Comment by mxmar** [comment]
Another thing is about PR look. Where is assigne, label and project?

File: go.mod [hunk 2]
```diff
@@ -69,66 +71,68 @@ require (
...
 )
 
-replace github.com/ethereum/go-ethereum => github.com/prysmaticlabs/bazel-go-ethereum v0.0.0-20210311182818-b9c0d847ae7b
-
-replace github.com/json-iterator/go => github.com/prestonvanloon/go v1.1.7-0.20190722034630-4f2e55fcf87b
-
-replace github.com/prysmaticlabs/prysm => github.com/lukso-network/vanguard-consensus-engine v0.0.1-0.20210322204411-9ca9f8244a4f
+replace (
+	github.com/ethereum/go-ethereum => github.com/prysmaticlabs/bazel-go-ethereum v0.0.0-20210707101027-e8523651bf6f
+	github.com/ferranbt/fastssz => github.com/atif-konasl/fastssz v0.0.0-20210921140807-706ef3f49459
```
Comment (mxmar) [inline go.mod:133]: What about this? We stay with your repo here?

Comment (atif-konasl) [reply go.mod:133]: Does it create any problem?

**Pull request merged**